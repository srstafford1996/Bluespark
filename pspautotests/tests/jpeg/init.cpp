#include <common.h>
#include <pspjpeg.h>
#include <psputility.h>

extern "C" int main(int argc, char *argv[]) {
	checkpointNext("Modules:");
	checkpoint("  LoadAv: %08x", sceUtilityLoadAvModule(PSP_AV_MODULE_AVCODEC));
	checkpoint("  UnloadAv: %08x", sceUtilityUnloadAvModule(PSP_AV_MODULE_AVCODEC));
	checkpoint("  Load: %08x", sceUtilityLoadModule(PSP_MODULE_AV_AVCODEC));

	checkpointNext("Sequence:");
	checkpoint("  Init: %08x", sceJpegInitMJpeg());
	checkpoint("  Twice: %08x", sceJpegInitMJpeg());
	checkpoint("  Finish: %08x", sceJpegFinishMJpeg());
	checkpoint("  After finish: %08x", sceJpegInitMJpeg());

	return 0;
}