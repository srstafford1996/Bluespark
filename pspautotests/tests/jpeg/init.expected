[x] Modules:
[r]   LoadAv: 00000000
[r]   UnloadAv: 00000000
[r]   Load: 00000000

[x] Sequence:
[r]   Init: 00000000
[x]   Twice: 80650042
[r]   Finish: 00000000
[r]   After finish: 00000000
